### PR TITLE
Fix bioholders not properly copying blood DNA, fingerprints, and genetic effects

### LINKED ISSUE #334: Stable Mutagen, and thus likely ling DNA sting, doesn't copy your fingerprints and blood DNA as it should according to the code.
[BUG]

**Describe the bug**
Stable Mutagen appears to not change your fingerprints or blood DNA, which the code of stable mutagen suggests it should, because it copies your bio-holder, which should also copy that over.

**To Reproduce**
1. Scan your person with a forensics scanner.
2. Use a stable mutagen mix on yourself with another players blood, so that you change.
3. Scan again with forensics scanner to see that you have failed to change your forensics.

**Expected behavior**
Stable Mutagen and Ling Sting should copy your fingerprints and blood DNA, as the code seems to suggest at a glance.

**Screenshots**

**Additional context**
Here's the relevant code for the stable mutagen behavior	
```
M.bioHolder.StaggeredCopyOther(data, progress_timer+=(1 * mult))
					if (progress_timer > 10)
						if (data_as_holder && data_as_holder.ownerName)
							M.real_name = data_as_holder.ownerName
							M.bioHolder.ownerName = M.real_name
							M.bioHolder.CopyOther(data)
						else if (data && data:ownerName)
							M.real_name = data:ownerName
							M.bioHolder.ownerName = M.real_name
```
```

### PR DESCRIPTION
**Pull request by flrsh**
[FIX] [ENHANCEMENT]

## About the PR 

This PR fixes #334; bioholders not properly copying fingerprints or genetic effects. It also cleans up stable mutagen code and makes transfer_blood slightly better (reduces bioholder creation + copying). Fixing these bugs restores functionality that was previously not ingame - mainly copying latent + active genetic effects.

## Why's this needed? 

Fixes a bug.

## Changelog

```
(u)Flourish:
(*)Cloning and stable mutagen now properly copy blood DNA, fingerprints, and genetic effects. The advanced genetic analysis option on the cloning computer toggles copying active genetic effects (yes, that includes mutantraces).
```

### CODE REVIEW

**Comment by Enakai** [comment]
Awesome, always good to see bugs fixed that inadvertently removed features. Will this also fix the problem with the lawgiver sometimes not carrying across the fingerprint ID after cloning?

**Comment by flrsh** [comment]
> Will this also fix the problem with the lawgiver sometimes not carrying across the fingerprint ID after cloning?

Yep, it will!

File: code/obj/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -251,8 +251,8 @@
 		if(5) //Advanced genetics analysis
 			dat += {"<h4>Advanced Genetic Analysis</h4>
 					<a href='byond://?src=\ref[src];menu=1'>Back</a><br>
-					<B>Notice:</B> Enabling this feature will prompt the attached clone pod to analyze the genetic makeup of the subject during cloning.
-					Data will then be sent to any nearby GeneTek scanners and be used to improve their efficiency. The cloning process will be slightly slower as a result.<BR><BR>"}
+					<B>Notice:</B> Enabling this feature will prompt the attached clone pod to transfer active genetic mutations from the genetic record to the subject during cloning.
```
Comment (Tarmunora) [inline code/obj/machinery/computer/cloning.dm:254]: Does it only copy active, or does it also copy potential?

Comment (flrsh) [reply code/obj/machinery/computer/cloning.dm:254]: Potential is always copied, active is toggleable with this.

**Pull request merged**